### PR TITLE
fix: endless login

### PR DESCRIPTION
**Pull request by ferferga**

### CODE REVIEW

File: components/Forms/LoginForm.vue [hunk 1]
```diff
@@ -102,8 +102,15 @@ export default Vue.extend({
 
       this.loading = true;
       this.setDeviceProfile();
-      await this.loginRequest(this.login);
-      this.loading = false;
+      try {
+        await this.loginRequest(this.login);
+        this.$router.replace('/');
```
Comment (MrTimscampi) [inline components/Forms/LoginForm.vue:107]: Why replace instead of push?

File: pages/wizard.vue [hunk 2]
```diff
@@ -124,7 +124,7 @@ export default Vue.extend({
       try {
         await this.$api.startup.completeWizard();
         // Redirect to setup complete page
-        this.$router.push('/login');
+        this.$router.replace('/login');
```
Comment (MrTimscampi) [inline pages/wizard.vue:127]: Same here?

File: components/Forms/LoginForm.vue (hunk 1 above)
Comment (ferferga) [reply components/Forms/LoginForm.vue:107]: So there's no way to go back. Doesn't make much sense if you're already logged in.

File: pages/wizard.vue (hunk 2 above)
Comment (ferferga) [reply pages/wizard.vue:127]: Same as above

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**